src/components/drawers/Analysis.vue

```diff
@@ -750,7 +757,8 @@ export default {
             let wins2 = move.black;
             let totalGames = wins1 + wins2;
```

Comment:
I thought about that but 
1. I thought it would misrepresent the winning chances for black. (not 100% sure that's true though)
2. if we don't display the draws, it might confuse the user when the ui says shows `50 white wins, 50 black wins, 105 games`

I was also considering displaying the draw-count directly, but because it's usually only a very small number of games I don't think there's much value to it

![image](https://github.com/gruppler/PTN-Ninja/assets/8362046/884cc3a6-0640-463e-a737-0ce014fbb3a2)

Reply:
Good point. Leaving it as it is would definitely be easiest, and possibly most practical. I do like the idea of showing draws and including them in the game counts, but that would seem to require more work than it's worth, and might not even be useful or desirable. Let's keep it as it is.

Reply:
It's not much work, if you like I'll create a PR for it - I'm just not sure if it helps the user or just clutters the UI.

Reply:
I don't think it's useful enough to surface. Plus, I'm not sure what theme colors you'd use for the fg/bg.